forage-android/src/main/java/com/joinforage/forage/android/core/element/state/PanElementStateManager.kt

```diff
@@ -106,6 +106,10 @@ class PanElementStateManager(state: ElementState, private val validators: Array<
         onChangeEventListener?.invoke(getState())
     }
 
+    fun canTokenizePanElementValue(alwaysAllow: Boolean = false) : Boolean {
```

Comment:
Saw that this was removed later, but just want to confirm that we definitely don't want to be restricting the submit function on our side. Let the user of the SDK make that decision.

Reply:
We're 100% on the same page! This method's only reference is `ForagePANEditText.shouldTokenize`, which has been [unused since PR 72](https://github.com/teamforage/forage-android-sdk/pull/72/files#diff-a78535f5e62364effd45eeb5e9b9abc64306d9e92f769f380195b6c359171306L45). As you pointed out, one of the PRs in this stack, we delete this method since it has no use. Let me know if that clarifies!